roles/client_config/tasks/install-rpk.yml

```diff
@@ -0,0 +1,29 @@
+- name: Set architecture fact for url
+  set_fact:
+    rpk_arch: "{{ 'amd64' if ansible_architecture == 'x86_64' else 'arm64' if ansible_architecture == 'aarch64' else 'unsupported' }}"
+
+- name: Ensure unzip is installed
+  ansible.builtin.package:
+    name: unzip
+    state: present
+  become: true
+
+- name: Download Redpanda rpk zip for amd64
+  ansible.builtin.get_url:
+    url: "https://github.com/redpanda-data/redpanda/releases/latest/download/rpk-linux-{{ rpk_arch }}.zip"
+    dest: "/tmp/rpk-linux-{{ rpk_arch }}.zip"
+    mode: '0644'
+  when: rpk_arch in ['amd64', 'arm64']
+
+- name: Unzip Redpanda rpk to /usr/local/bin
+  ansible.builtin.unarchive:
+    src: "/tmp/rpk-linux-{{ rpk_arch }}.zip"
+    dest: /usr/local/bin
+    remote_src: yes
+  become: yes
+  when: rpk_arch in ['amd64', 'arm64']
+
+- name: Fail for unsupported architecture
+  fail:
+    msg: "The architecture '{{ ansible_architecture }}' is not supported for Redpanda rpk installation"
+  when: rpk_arch == 'unsupported'
```

Comment:
for short term, I guess this is fine, but we really need to have a package-isntallable version of rpk @deniscoady. 

Personally, I'd say just install the redpanda package and for client nodes make sure redpanda.service is turned off in systemd. 

This also puts us back in the situation of having a difficult airgap setup because it's basically enabling the same type of always-on-the-internet behavior that the repo setup scripts did.

Reply:
btw, at least on rpm-based stuff, getting that package built should be pretty straight forward. Some modifications of the specfile would do it. I suspect similar would happen in deb builds.

Reply:
I wanted to keep client very basic and light, so I didn't do a full RP install. I also assumed that client is something that has some external access (jumpbox/proxy/etc) so should be able to handle grabbing the file. For instance in the playbooks that use the client role it's on the same system as the squid proxy.

Reply:
Let's use the package for everything. We already include rpk in there. This solves a bunch of the hoop jumping as well since we already handle the rpms/debs elsewhere. We're not hurting for disk space on the clients. And it solves keeping the rpk/redpanda versions kind of in sync (and other upgrade issues)

If we want a lighter deployment, we should fix the package so it splits the to a -client and -server component.

Reply:
Either approach seems fine,
we're quickly coming to a point where we need to separate redpanda and rpk installs.
Since the client already is installing a proxy, is there a downside to having a direct package install? Note that a direct package install is necessary for mac as well, since we don't have releases for darwin.

Reply:
I'm going to keep the unzip package from github release approach. It seems really odd to go through the effort of installing the full RP binary just for RPK when we have a functional alternative. Especially when the likelihood of end users using this client is low. This is lightweight and easily ripped and replaced when the CLI and RP repositories are split in dnf/apt.